tests/test_action_checks.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python3
+
+import unittest
+import re
+import sys
+
+class TestActionChecks(unittest.TestCase):
+    def setUp(self):
+        self.review_pattern = r"\b((?:no|skip)-(?:review|otter|otterai)|otter-(?:no|bye|restricted))(?:,((?:no|skip)-(?:review|otter|otterai)|otter-(?:no|bye|restricted)))*\b"
+        self.pr_state_pattern = r"\b(?:merged|closed)\b"
+
+    def check_review_requested(self, pr_title):
+        """Simulate the 'Check if review is requested' step"""
+        if re.search(self.review_pattern, pr_title, re.IGNORECASE):
```

Comment:
⚠️ Instead of printing, consider using logging for better control over output. 📜